File: clients/typescript/examples/client-get-db-admin-from-db.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

// Get a database object
const client = new DataAPIClient();
const database = client.db("**API_ENDPOINT**", {
  token: "**APPLICATION_TOKEN**",
});

// Get a database admin object
const databaseAdmin = database.admin();

File: clients/java/examples/client-get-db-admin-from-db.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.admin.DatabaseAdmin;
import com.datastax.astra.client.databases.Database;

public class Example {

  public static void main(String[] args) {
    // Get a database object
    Database database = new DataAPIClient("**APPLICATION_TOKEN**").getDatabase("**API_ENDPOINT**");

    // Get a database admin object
    DatabaseAdmin databaseAdmin = database.getDatabaseAdmin();
  }
}
